CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Replace class-based `Grid` with StyledComponent alternative
```

Comment:
Si le composant a changé de chemin, est-ce que ce n'est pas un breaking change ?

Reply:
Le chemin aurait été le même, mais je l'ai renommé parce que je ne pense plus que c'est pertinent de remplacer le `Grid` original

Reply:
Du coup, on pourrait réécrire l'entrée dans le `CHANGELOG` : 
```
Feature: Add a new component `AdaptableGrid` to handle grids. From now, you can use this component instead of `Grid`.
```